src/main/java/ch/njol/skript/conditions/CondIsDivisibleBy.java

```diff
@@ -0,0 +1,64 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Checker;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Divisible By")
+@Description("Check if a number is divisible by another number.")
+@Examples({
+	"if 5 is divisible by 5:",
+	"if 11 cannot be divided by 10:",
+})
+@Since("INSERT VERSION")
+public class CondIsDivisibleBy extends Condition {
+
+	static {
+		Skript.registerCondition(CondIsDivisibleBy.class,
+			"%numbers% (is|are) evenly divisible by %number%",
+			"%numbers% (isn't|is not|aren't|are not) evenly divisible by %number%",
+			"%numbers% can be evenly divided by %number%",
+			"%numbers% (can't|cannot|can not) be evenly divided by %number%");
+	}
+
+	@SuppressWarnings("null")
+	private Expression<Number> divisorExpression;
+	@SuppressWarnings("null")
+	private Expression<Number> dividendExpression;
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		divisorExpression = (Expression<Number>) exprs[0];
+		dividendExpression = (Expression<Number>) exprs[1];
```

Comment:
```suggestion
		dividendExpression = (Expression<Number>) exprs[0];
		divisorExpression = (Expression<Number>) exprs[1];
```
These need to be flipped around.